es73x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala

```diff
@@ -178,7 +178,7 @@ class ReadonlyRestPlugin(s: Settings, p: Path)
                                indexNameExpressionResolver: IndexNameExpressionResolver,
                                nodesInCluster: Supplier[DiscoveryNodes]): util.List[RestHandler] = {
     List[RestHandler](
-      new RestRRAdminAction(settings, restController)
+      new RestRRAdminAction(restController)
```

Comment:
I don't agree with this change either. I see that implementation of BaseRestHandler in ES 7.3.2 has "todo" in base constructor, but:
* we have to be sure that all versions, starting from ES 7.0.0 to ES 7.3.2, don't use the settings
* passing null as settings strongly relays on knowing the implementation details, not on interface and documentation

Please, back to previous version.

Reply:
You're right. I forgot to ensure versions before 7.3.2.